### PR TITLE
qa/verification-framework-changes-part-2 (ENG-1067)

### PR DESCRIPTION
**Pull request by jonathanprozzi**
## Linear Ticket

- [ENG-1067](https://linear.app/govrn/issue/ENG-1067/vf-changes-prior-to-release-part-2)

## Description

- Addresses the feedback from QA

## Screencaptures

Coming soon -- will add when this moves out of draft

### CODE REVIEW

File: apps/protocol-frontend/src/components/AttestationsTable.tsx [hunk 1]
```diff
@@ -135,18 +135,21 @@ const AttestationsTable = ({
...
-          const currentAttestations = row.original.attestations?.length;
           const frameworkSettingThreshold =
             row.original.guilds[0].guild.verification_setting
               ?.num_of_attestations;
           let pillUnverifiedLabel!: string;
           if (attestationThreshold && frameworkSettingThreshold) {
-            pillUnverifiedLabel = `${
-              frameworkSettingThreshold - currentAttestations
-            }/${frameworkSettingThreshold}`;
+            pillUnverifiedLabel = `${currentAttestations}/${frameworkSettingThreshold}`;
```
Comment (Flip-Liquid) [inline apps/protocol-frontend/src/components/AttestationsTable.tsx:146]: sweet

Comment (jonathanprozzi) [reply apps/protocol-frontend/src/components/AttestationsTable.tsx:146]: I know you caught that yesterday but I didn't remove the frameworkThreshold from the first part so it wasn't accurate!

File: apps/protocol-frontend/src/components/AttestationsTable.tsx [hunk 2]
```diff
@@ -135,18 +135,21 @@ const AttestationsTable = ({
...
+          }
+
+          if (status === 'Unverified' && attestationThreshold === null) {
```
Comment (Flip-Liquid) [inline apps/protocol-frontend/src/components/AttestationsTable.tsx:149]: what about DAOs that have not adopted a threshold? do we still want to call this initializing?

Comment (jonathanprozzi) [reply apps/protocol-frontend/src/components/AttestationsTable.tsx:149]: We have that covered as 'Not Set' for now

File: apps/protocol-frontend/src/components/AttestationsTable.tsx [hunk 3]
```diff
@@ -157,25 +160,48 @@ const AttestationsTable = ({
...
           }
 
           let pillStatusMap!: 'checkmark' | 'secondaryInfo' | 'primaryInfo';
+
           if (status === 'Verified' || frameworkSettingThreshold === 0) {
             pillStatusMap = 'checkmark';
           }
+
+          if (
+            attestationThreshold === null &&
```
Comment (Flip-Liquid) [inline apps/protocol-frontend/src/components/AttestationsTable.tsx:182]: nit: consistent ordering with the conditional on 164

**Review by Flip-Liquid — APPROVED** [review]
just a few nonblocking nits, otherwise LGTM

File: apps/protocol-frontend/src/components/AttestationsTable.tsx (hunk 3 above)
Comment (jonathanprozzi) [reply apps/protocol-frontend/src/components/AttestationsTable.tsx:182]: Thanks! good call!

**Comment by jonathanprozzi** [comment]
> 

awesome thanks, merging in staging with the updates and pushing up a fix to address feedback

**Pull request merged**